R/install.R

```diff
@@ -79,22 +79,15 @@ py_install <- function(packages,
   # then install into that same environment
   if (is.null(envname)) {
     
-    python <- if (is_python_initialized())
-      .globals$py_config$python
-    else if (length(.globals$required_python_version))
-      .globals$required_python_version[[1]]
-    else
-      NULL
+   config <- py_discover_config()
```

Comment:
I think we could still use `.globals$py_config$python` if Python is indeed already initialized.